app/scripts/jsx/EnhancedFixedDatatable.jsx

```diff
@@ -397,6 +397,11 @@ var EnhancedFixedDataTable = (function() {
 // Table prefix component
 // Contains components above the main part of table
   var TablePrefix = React.createClass({
+    removeResetButton: function () {
+      var resetSpan = this.getElementsByClassName("EFDT-header-filters-reset").item(0);
+      resetSpan.parentNode.parentNode.removeChild(resetSpan.parentNode);
```

Comment:
@stefanches7 you should conditionally render `removeResetButton` similar to what we have right now `this.props.filteredRowsSize !== this.props.rowsSize`